apps/camera/js/views/controls.js

```diff
@@ -245,11 +250,19 @@ module.exports = View.extend({
     this.unset(key ? key + '-enabled' : 'enabled');
   },
 
+  localize: function() {
+    for (var el in this.elsL10n) {
+      this.els[el].setAttribute('data-l10n-id', this.elsL10n[el]);
```

Comment:
There are a couple of things here, I think:
* l10n is not ready when the template is loaded and it does not get localized in time. localize for controls will be called on app localized event
* there is another PR on its way with localization for the labels for the rest of the controls, so I just wanted to keep it all together. I can do it without the loop as well.